service/src/main/java/bio/terra/workspace/common/utils/ControllerValidationUtils.java

```diff
@@ -36,6 +44,9 @@ public final class ControllerValidationUtils {
   public static final Pattern USER_FACING_ID_VALIDATION_PATTERN =
       Pattern.compile("^[a-z0-9][-_a-z0-9]{2,62}$");
 
+  public static final Map<String, Set<String>> REQUIRED_POLICY_INFORMATION =
+      ImmutableMap.of("terra:group-constraint", ImmutableSet.of("group-name"));
```

Comment:
I am changing this to "groups", because it is going to be a comma-separated list of groups.
But as Mike suggests, this validation should probably be in TPS.
Do we need a validatePao endpoint or should it be a side-effect of calling any of Pao-receiving TPS endpoints?

Reply:
It would require an (internal for now, external later) service call but save us from launching + undoing a flight, so it's probably a performance improvement but not required for functionality